src/main/java/net/sf/mzmine/modules/peaklistmethods/peakpicking/deconvolution/DeconvolutionTask.java

```diff
@@ -59,6 +61,11 @@
     // User parameters
     private final ParameterSet parameters;
 
+	private RSessionWrapper rSession;

```

Comment:
To be honest, I liked the previous solution better. Adding more methods to the PeakResolver interface just complicates things.. and those Object... args look rather confusing. 

I will merge the branch as it is now :)